### ISSUE TITLE
Shutdown bug

### ISSUE THREAD

**Issue by hexaredecimal** [hexaredecimal/BlazingWebX#8]
Encountered the following bug on the deployment platform when the server spins down. 
```
Exception in thread "Thread-1" java.lang.IllegalStateException: Shutdown in progress
	at java.base/java.lang.ApplicationShutdownHooks.remove(Unknown Source)
	at java.base/java.lang.Runtime.removeShutdownHook(Unknown Source)
	at blazing.Blazing$1.run(Blazing.java:246)
```
This is an error related to the shutdown hook used for destructuring objects. The exception is thrown because the reference to the hook tries to remove it's self from the shutdown hooks inside its own thread. 
https://github.com/hexaredecimal/BlazingWebX/blob/d1727ddc4c9ccbbb8eb6f9f073cffbe804ae50f1/src/blazing/Blazing.java#L246C6-L246C52

**Comment by hexaredecimal** [comment]
Fixed on this commit: [d7a4aef](https://github.com/hexaredecimal/BlazingWebX/commit/d7a4aef952fdc6519b5f6fb9bb742a1fd176578a)

**Comment by hexaredecimal** [comment]
Fixe on this latest commit. e058d58
